bower.json

```diff
@@ -9,7 +9,10 @@
     "angular-route-segment": "1.5.1",
     "angular-bootstrap": "2.5.0",
     "bootstrap": "3.3.7",
-    "bootstrap-sass": "3.3.7"
+    "bootstrap-sass": "3.3.7",
+    "metisMenu": "metismenu#^2.7.0",
+    "jquery-slimscroll": "jquery.slimscroll#^1.3.8",
+    "font-awesome": "Font-Awesome#^4.7.0"
```

Comment:
@zpawn Don't forget to remove version sign (^). At some time module will be updated, and we will have a problems. Please, remove all ^ signs.